layers/meta-resin-artik/recipes-bsp/bcm4354-bluetooth/bcm4354-bt.bbappend

```diff
@@ -0,0 +1,15 @@
+FILESEXTRAPATHS_append := ":${THISDIR}/files"
+
+# add patch file that mainly moves the bluetooth mac address file into the resin-conf partition (needed for not clashing with resinhup)
```

Comment:
This description should be in the patch header and removed from here. Good content anyway.